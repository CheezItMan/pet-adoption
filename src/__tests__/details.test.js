import React from "react";
import { create } from "react-test-renderer";
import Details from "../components/details";

test("snapshot", () => {
  const component = create(<Details />);

  expect(component.toJSON()).toMatchSnapshot();
});

test("shows model when toggleModel is called", () => {
  const component = create(<Details search={() => {}} />);

  const instance = component.getInstance();

  expect(instance.state.showModal).toBe(false);
  instance.toggleModal();
  expect(instance.state.showModal).toBe(true);
});
