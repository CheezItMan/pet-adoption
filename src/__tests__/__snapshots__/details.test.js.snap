// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div>
  <h1>
    Loading
  </h1>
</div>
`;
